Sandbox/Sandbox.xcodeproj/project.pbxproj

```diff
@@ -497,19 +519,36 @@
 		};
 		3417320D229D6B6400681CFD /* Debug */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = 81035E8DEC67D32DB84E55AA /* Pods-Sandbox.debug.xcconfig */;
+			baseConfigurationReference = 599E4AA754038D29E0738EAC /* Pods-Sandbox.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = 6KDQ68PRPG;
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/Alamofire\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/BrightFutures\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/CryptoSwift\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/FBSDKCoreKit\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/FBSDKLoginKit\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/FacebookCore\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/FacebookLogin\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/GTMSessionFetcher\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/GoogleToolboxForMac\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/IdentitySdkCore\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/IdentitySdkFacebook\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/IdentitySdkGoogle\"",
+					"\"${PODS_CONFIGURATION_BUILD_DIR}/IdentitySdkWebView\"",
+					"\"${PODS_ROOT}/GoogleSignIn/Frameworks\"",
+				);
 				INFOPLIST_FILE = Sandbox/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 12.2;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.reach5.Sandbox;
+				PRODUCT_BUNDLE_IDENTIFIER = com.reach5.Sandbox2;
```

Comment:
You needed to create a second Sandbox in order to test it?

Reply:
i used Sandbox2 for testing with device because we haven't yet a team certificate. but i will change it